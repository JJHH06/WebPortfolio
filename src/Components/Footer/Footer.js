import React from 'react';

function Footer(){

    return ( 
        <div className="page-footer text-white-50 footer-portafolio">
        <br/><p>&nbsp;&nbsp;&nbsp;&nbsp;Â© Jose Hurtarte  <p className="text-white">&nbsp;&nbsp;&nbsp;&nbsp;hur19707@uvg.edu.gt</p></p>
      </div>

        
    );
}

export default Footer;