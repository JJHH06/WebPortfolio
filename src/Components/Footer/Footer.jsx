import React from 'react';

function Footer() {
  return (
    <div data-testid="footer" className="page-footer text-white-50 footer-portafolio">
      <br />

&nbsp;&nbsp;&nbsp;&nbsp;Â© Jose Hurtarte
      <p className="text-white">&nbsp;&nbsp;&nbsp;&nbsp;hur19707@uvg.edu.gt</p>

    </div>

  );
}

export default Footer;
