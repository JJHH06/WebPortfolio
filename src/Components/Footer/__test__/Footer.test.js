/**
 * @jest-environment jsdom
 */
/* eslint-disable*/
import React from 'react';
import ReactDOM from 'react-dom';
import { cleanup } from '@testing-library/react';
import { isTSanyKeyword } from '@babel/types';
import Footer from '../Footer.jsx';
import {render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from "react-test-renderer";


afterEach(cleanup);
it('renders footer correctly', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Footer />, div);
});

it("Renders footer text correctly",()=>{
    const {getByTestId} = render(<Footer/>)
    expect(getByTestId('footer')).toHaveTextContent("Â© Jose Hurtarte")
})

it("matches snapshot",()=>{
    const tree = renderer.create(<Footer/>).toJSON()
    expect(tree).toMatchSnapshot();
})