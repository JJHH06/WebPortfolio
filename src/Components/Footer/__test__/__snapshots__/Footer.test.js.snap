// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="page-footer text-white-50 footer-portafolio"
  data-testid="footer"
>
  <br />
      © Jose Hurtarte
  <p
    className="text-white"
  >
        hur19707@uvg.edu.gt
  </p>
</div>
`;
